package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodist",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
May want to also be updating the version that appears in windows: https://github.com/nullivex/nodist/blob/master/build/Nodist.template.nsi#L10
Currently, in `0.7.1` it still says `0.7.0.0` so it was missed for that version.